srcpkgs/liquid-dsp/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'liquid-dsp'
```

Comment:
This package doesn't have a versioned shared library

Reply:
I added patch from Debian to fix this.